sjb/config/test_cases/test_pull_request_openshift_ansible_extended_conformance_install.yml

```diff
@@ -1,6 +1,7 @@
 ---
 parent: 'test_cases/test_branch_origin_extended_conformance_install.yml'
 overrides:
+  junit_analysis: True
```

Comment:
no

Reply:
don't we want this one to have junit analysis?

Reply:
@stevekuznetsov `test_pull_request_openshift_ansible_extended_conformance_install_*.yml` need `junit_analysis: True` because their parent template `test_cases/test_branch_origin_extended_conformance_install.yml` needs `junit_analysis: False`

Reply:
oh these are trues...

Reply:
🌮